src/Raven.Server/Documents/DocumentIdWorker.cs

```diff
@@ -404,6 +404,15 @@ public static void ThrowDocumentIdTooBig(string str)
                 nameof(str));
         }
 
+        public static void ThrowDocumentIdTooBig(string id, string changeVector)
+        {
+            throw new ArgumentException(
+                $"Revision change vector cannot exceed {MaxIdSize} bytes, but the change vector was {Encoding.GetByteCount(changeVector)} bytes. " +
+                $"The invalid change vector for revision '{id}' is '{changeVector}'.{Environment.NewLine}" +
+                $"For more details visit https://ravendb.net/l/ZH9AK4/5.4.{Environment.NewLine}",
```

Comment:
note @ppekrol that we need to update this to the real link

Reply:
put a space between last dot and env variable. Have a feeling that most of the url parsers will not catch that

Reply:
done